packages/SwingSet/src/liveslots/virtualObjectManager.js

```diff
@@ -411,6 +449,27 @@ export function makeVirtualObjectManager(
     }
   }
 
+  function copyMethods(behaviorTemplate) {
+    const obj = {};
+    for (const [name, func] of Object.entries(behaviorTemplate)) {
+      assert.typeof(func, 'function');
+      obj[name] = func;
+    }
+    return obj;
+  }
+
+  function bindMethods(context, behaviorTemplate) {
+    const obj = {};
+    for (const [name, func] of Object.entries(behaviorTemplate)) {
+      assert.typeof(func, 'function');
+      const method = (...args) => Reflect.apply(func, null, [context, ...args]);
+      unweakable.add(method);
+      unweakable.add(context);
```

Comment:
I'd suggest adding `context` to the unweakables down where it's created, rather than here (multiple times).